modules/cert-manager-override-flag-controller.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-flag-controller_{context}"]
+= Customizing the cert-manager controller by overriding flags
+
+You can enable the `--enable-certificate-owner-ref` flag for the {cert-manager-operator} by adding a `spec.controllerConfig` section in the `CertManager` resource.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have installed the {cert-manager-operator}.
+
+
+.Procedure
+
+. Check that the `ClusterIssuer` object and its secret is available by running the following command:
++
+[source,terminal]
+----
+$ oc get cert
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                             READY   SECRET                                           AGE
+certificate-from-clusterissuer-route53-ambient   True    certificate-from-clusterissuer-route53-ambient   8h
+----
+
+. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager cluster
+----
+
+. Add a `spec.controllerConfig` section with the following override arguments:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+  ...
+spec:
+  ...
+  controllerConfig:
+    overrideArgs:
+      - '--enable-certificate-owner-ref' <1>
+----
+<1> The `--enable-certificate-owner-ref` flag sets the certificate resource as an owner of the secret where the TLS certificate is stored.
++
+[WARNING]
+====
+If you uninstall the cert-manager Operator, cert-manager controller, CA injector, Webhook, or certificate resource, the secret is deleted automatically. This might cause network connectivity issues.
```

Comment:
```suggestion
If you uninstall the {cert-manager-operator} or delete the certificate resource(s) from the cluster, the secret is deleted automatically. This might cause network connectivity issues depending upon where the certificate TLS secret is being used.
```